test/email_alert_api_test.rb

```diff
@@ -323,6 +323,40 @@
         end
       end
     end
+
+    describe "when content_id and links are provided" do
+      let(:content_id) { "fd427f55-7492-440d-ae3f-0fb6c3592b21" }
+      let(:slug) { "my new list" }
```

Comment:
Nitpick: This isn't a slug!

Reply:
It isn't you're quite right. I will fix!